### PR TITLE
Add missing PHPDoc param type

### PR DESCRIPTION
**Pull request by ntzm**

### CODE REVIEW

File: src/Fixer/Import/OrderedImportsFixer.php [hunk 1]
```diff
@@ -474,7 +474,7 @@ private function getNewOrder(array $uses, Tokens $tokens)
     }
 
     /**
-     * @param $indexes
+     * @param array $indexes
```
Comment (keradus) [inline src/Fixer/Import/OrderedImportsFixer.php]: if we are doing this, array of what ?

Comment (keradus) [reply src/Fixer/Import/OrderedImportsFixer.php]: typehinting in method header is missing then

File: src/Fixer/Import/OrderedImportsFixer.php [hunk 2]
```diff
@@ -474,11 +474,11 @@ private function getNewOrder(array $uses, Tokens $tokens)
     }
 
     /**
-     * @param $indexes
+     * @param array[] $indexes
```
Comment (keradus) [inline src/Fixer/Import/OrderedImportsFixer.php]: wow, this is array of arrays? really? what's inside? could you give example?

Comment (ntzm) [reply src/Fixer/Import/OrderedImportsFixer.php]: They are built here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/760f404720db334e4a52bde5daf83123ea152f2e/src/Fixer/Import/OrderedImportsFixer.php#L410-L416

Comment (SpacePossum) [reply src/Fixer/Import/OrderedImportsFixer.php]: the alternative to avoid this was `continue 2;` IIRC ;)

**Comment by SpacePossum** [comment]
is fabbot outdated? I can't recall how the `for (;;)` should look like in SF style

File: src/Fixer/Import/OrderedImportsFixer.php (hunk 2 above)
Comment (keradus) [reply src/Fixer/Import/OrderedImportsFixer.php]: no, solution is to use real DTO and never use assoc arrays

**Comment by keradus** [comment]
behaviour of rule was modified on higher branch to handle this case better, and on 2.2 we kept code matching old behaviour, as we are running fixer as well, so better to have failing fabbot than fixer from our perspective

**Comment by keradus** [comment]
Thank you @ntzm.

**Pull request closed without merging**